src/component/archive/ArchiveTable/FeatureArchivedCell/FeatureArchivedCell.tsx

```diff
@@ -0,0 +1,36 @@
+import { VFC } from 'react';
+import TimeAgo from 'react-timeago';
+import { Tooltip, Typography } from '@mui/material';
+import { formatDateYMD } from 'utils/formatDate';
+import { TextCell } from 'component/common/Table/cells/TextCell/TextCell';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+
+interface IFeatureArchivedCellProps {
+    value?: string | Date | null;
+}
+
+export const FeatureArchivedCell: VFC<IFeatureArchivedCellProps> = ({
+    value: archivedAt,
+}) => {
+    const { locationSettings } = useLocationSettings();
+
+    if (!archivedAt) return <TextCell />;
+
+    return (
+        <TextCell>
+            {archivedAt && (
+                <Tooltip
+                    title={`Archived on: ${formatDateYMD(
+                        archivedAt,
+                        locationSettings.locale
+                    )}`}
+                    arrow
+                >
+                    <Typography noWrap variant="body2" data-loading>
+                        <TimeAgo date={new Date(archivedAt)} />
```

Comment:
To make this behave correctly in terms of UI/UX and keep it consistent with other uses of this component, I suggest adding the props: `title="" live={false}`.